pkg/cri/server/container_stop.go

```diff
@@ -73,6 +74,12 @@ func (c *criService) StopContainer(ctx context.Context, r *runtime.StopContainer
 func (c *criService) stopContainer(ctx context.Context, container containerstore.Container, timeout time.Duration) error {
 	id := container.ID
 
+	sandboxID := container.SandboxID
+	sb, err := c.sandboxStore.Get(sandboxID)
+	if err != nil {
+		return fmt.Errorf("can't find sandbox for sandbox ID %s: %w", sandboxID, err)
```

Comment:
same here .. not being able to stop the container because you can't find the podsandbox seems to be a new rule to the cri api that don't need to enforce

would rather be able to continue on and stop/remove a container that does not have a pod wrapper than force the leak of the container not being able to be stopped and removed when somehow the pod is no longer in the store...

Reply:
Done